agent/consul/config_endpoint_test.go

```diff
@@ -1029,10 +1029,12 @@ func TestConfigEntry_ResolveServiceConfig(t *testing.T) {
 			"foo": 1,
 		},
 	}))
+	meta := map[string]string{"foo": "bar"}
```

Comment:
Probably clearer here to just inline this in both places. That also insures that if the call to `EnsureConfigEntry` were to mutate this input, the test would fail.